arches_lingo/src/arches_lingo/components/scheme/report/SchemeNote.vue

```diff
@@ -1,20 +1,154 @@
+            detail:
+                error instanceof Error
+                    ? error.message
+                    : $gettext("Could not fetch the notes for the resource"),
+        });
+    }
+}
+
+async function deleteSectionValue(tileId: string) {
+    let result = false;
+    try {
+        result = await deleteSchemeNoteTile(route.params.id as string, tileId);
+    } catch (error) {
+        toast.add({
+            severity: ERROR,
+            summary: $gettext("Error"),
+            detail:
+                error instanceof Error
+                    ? error.message
+                    : $gettext("Could not delete selected note"),
+        });
+    }
+
+    if (result) {
+        getSectionValue();
+    }
+}
+
+function editSectionValue(tileId: string) {
+    const schemeStatement = schemeInstance.value?.statement?.find(
+        (tile) => tile.tileid === tileId,
+    );
+    if (schemeStatement && schemeStatement.tileid === tileId) {
+        emits(OPEN_EDITOR, schemeStatement.tileid);
+    } else {
+        toast.add({
+            severity: ERROR,
+            summary: $gettext("Error"),
+            detail: $gettext("Could not find the selected label to edit"),
+        });
+    }
+}
 </script>
 
 <template>
-    <SchemeReportSection :title-text="$gettext('Scheme Notes')">
-        abc
-    </SchemeReportSection>
+    <div v-if="mode === VIEW">
+        <SchemeReportSection
+            :title-text="$gettext('Scheme Notes')"
+            @open-editor="emits(OPEN_EDITOR)"
+        >
+            <MetaStringViewer
+                :meta-strings="schemeInstance?.statement"
+                :meta-string-text="metaStringLabel"
+                @edit-string="editSectionValue"
+                @delete-string="deleteSectionValue"
+            >
+                <template #name="{ rowData }">
+                    {{ (rowData as SchemeStatement).statement_content_n1 }}
+                </template>
+                <template #type="{ rowData }">
+                    <ControlledListItem
+                        :value="(rowData as SchemeStatement).statement_type_n1"
+                    />
+                </template>
+                <template #language="{ rowData }">
+                    <ControlledListItem
+                        :value="
+                            (rowData as SchemeStatement).statement_language_n1
+                        "
+                    />
+                </template>
+                <template #drawer="{ rowData }">
+                    <div>
+                        {{ $gettext("Bibliographic Sources") }}:
+                        <ResourceInstanceRelationships
+                            :value="
+                                (rowData as SchemeStatement)
+                                    .statement_data_assignment_object_used
+                            "
+                        />
+                    </div>
+                    <div>
+                        {{ $gettext("Contributors") }}:
```

Comment:
one small thing I missed first pass, mind wrapping these interpolations in a span or div or something? Prevents "naked" strings in the HTML